src/main/java/com/xceptance/neodymium/util/SelenideAddons.java

```diff
@@ -160,7 +160,8 @@ public ElementsCollection get()
             }
             catch (final Throwable t)
             {
-                if (isThrowableCausedBy(t, StaleElementReferenceException.class))
+                if (isThrowableCausedBy(t, StaleElementReferenceException.class)
+                    || t.getMessage().contains("Actual value: StaleElementReferenceException: stale element refe"))
```

Comment:
the isThrowableCausedBy method is recursively checking the Throwables to find a matching cause. Your method is only checking the latest message. Should we consider checking recursively for the message? 

Furthermore: We should think about improving the message we are searching for.  Maybe only "StaleElementReferenceException"?

Reply:
AFAIK, if SERE is thrown while retrieving the actual display status, it doesn't contain any other exceptions in the cause, so we probably don't need to check for the message recursively. Moreover, the message of exception in the cause is mostly contained in the original error message. 

A more broad search phase as  "StaleElementReferenceException" can be helpful in case there are some further variants of the SERE exception. On the other hand, the more specific the search phrase, the less is the probability to catch the "real" error (meaning not the SERE but an error about unexpected website behavior). However, I cannot think of any exception, which can contain the "StaleElementReferenceException" phrase but has nothing to do with SERE. So, I think, we can improve the search phrase.

Reply:
Search phrase improved